package.json

```diff
@@ -48,8 +48,8 @@
     "player"
   ],
   "dependencies": {
-    "video.js": ">=6.0.0",
-    "wavesurfer.js": ">=2.0.3"
+    "video.js": ">=6.8.0",
+    "wavesurfer.js": ">=2.0.5"
   },
```

Comment:
@hersche can you revert these changes? we don't depend on any newer version.

Reply:
shure, but i also tracked another issue in the testing-class..
you can also do a alternative solution to the counter, it's just workarounded so i can use it